library/packages/src/modules/PackageCallbacks.rb

```diff
@@ -215,13 +215,16 @@ def DoneProvide(error, reason, name)
         return "C"
       end
 
-      # INVALID
-      if error == 3
+      message =
+        if error == 3
```

Comment:
please use case here and add also default where you log unknown error code, so we know when something new appear.